### DISCUSSION TITLE
Welcome to GDSC-SMIT-App Discussions!

### DISCUSSION THREAD

**Discussion by Aryan-Lohia** [Aryan-Lohia/GDSC-SMIT-App#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayato91** [comment]
Hey Aryan! Nice to see you here. You are doing great. Keep-going :)
